src/_pytest/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 try:
     from ._version import version as __version__
-except ImportError:
+except ImportError:  # pragma: no cover
     # broken installation, we don't even try
     # unknown only works because we do poor mans version compare
     __version__ = "unknown"
```

Comment:
Better leave this code as uncovered.
We're not trying to hit 100%, but is is OK for this to be not covered - also for other things below.

Reply:
> We're not trying to hit 100%

We're not? 😆

Reply:
Confirmation that, in fact, _we are_:

https://github.com/pytest-dev/pytest/pull/2800#pullrequestreview-65593151

> Ultimately we should strive to obtain 100% coverage...

Reply:
/cc @nicoddemus 

I think the crucial part from there is:

> Even if we "cheat", we gain the benefit of demanding that new code from now on has 100% coverage.

This should be achieved instead by requiring a diff to be covered 100% - something which codecov provides then.